src/component/common/ConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/ConstraintAccordionEditBody.tsx

```diff
@@ -97,7 +97,7 @@ const InvertedOperator = ({
                         color="primary"
                     />
                 }
-                label={'negated'}
+                label="Negated"
```

Comment:
Silly thing but are we using ' or " in this project? The use of double quotes here stands out a little from the rest of the PR

Reply:
Prettier enforces double quotes for simple JSX props, though overridable. I suppose it's trying to mimic HTML.

Reply:
Ah, that makes sense!